# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../7zip/install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- p7zip_4.18/install.sh.orig	2005-04-26 20:15:53.000000000 +0200
+++ p7zip_4.18/install.sh	2005-05-12 23:32:31.000000000 +0200
@@ -1,7 +1,7 @@
 #! /bin/sh
 
 # global install
-DEST_HOME=/usr/local
+DEST_HOME=/usr
 # for testing 
 #DEST_HOME=${HOME}/INSTALL/usr/local
 DEST_BIN=${DEST_HOME}/bin
